decidim-bulletin_board-ruby/spec/fixtures/bb_schema.json

```diff
@@ -0,0 +1,1806 @@
+{
```

Comment:
I'm not sure if we need to check out this file in the repository yet 🤔  Let's leave it for now, but we should think about that in the future